EnglishFilter/sections/foreign.txt

```diff
@@ -7025,7 +6986,6 @@ dobryruch.co.uk#$#.adsbygoogle { position: absolute!important; left: -3000px!imp
 ! https://github.com/AdguardTeam/AdguardFilters/issues/53284
 ! https://github.com/AdguardTeam/AdguardFilters/issues/50437
 @@||js.iplsc.com/inpl.abd/*/proxy.html?host=$generichide
-@@||js.iplsc.com/inpl.abd/latest/ads.js?$domain=pomponik.pl|interia.pl|js.iplsc.com
```

Comment:
Cannot be removed. Matched by many rules.

Reply:
Yeah you're right, the path include lots of baits.